### PR TITLE
All options and more

### PR DESCRIPTION
**Pull request by sharevb**
Corrected almost all opened issues, added almost all options from docker run -h
Read all doc of Docker compose and Docker run
Also added ablility to merge with an existing compose file

Checklist
[x ] Added a test to cover all behaviours
[x ] Verified some commands and multiline support
[ x] Behaves according the documentation as far as understood

### CODE REVIEW

File: packages/composerize-website/src/components/Entry.js [hunk 1]
```diff
@@ -39,10 +39,23 @@ export default function Entry(props) {
...
                     </Blurb>
-                    <TextInput command={props.command} onInputChange={props.onInputChange} />
+                    <TextInput value={props.command} rows={3} onInputChange={props.onCommandInputChange} />
+                    <details>
+                        <summary>
+                            Click and paste to append <Code>docker run</Code> commands to an existing Docker compose
+                            file
+                        </summary>
+                        <TextInput value={props.compose} rows={10} onInputChange={props.onComposeInputChange} />
+                    </details>
```
Comment (magicmark) [inline packages/composerize-website/src/components/Entry.js:58]: what is this doing?

it also looks like it needs some margin on the bottom

Screenshot 2023-10-16 at 8 31 30 AM

File: README.md [hunk 2]
```diff
@@ -30,4 +78,5 @@ Needs yarn@1.19.1. See https://github.com/yarnpkg/yarn/issues/7734.
 
 ## Maintainers
 
+- ShareVB [GitHub](https://github.com/sharevb)
```
[the remark this answers is not in the document]
Comment (sharevb) [reply README.md:81]: ok, no problem, I should have added me second by the way, sorry
Should I remove line or add after you or any other text/section ? thanks

File: packages/composerize-website/src/components/Entry.js (hunk 1 above)
Comment (sharevb) [reply packages/composerize-website/src/components/Entry.js:58]: This allows to paste an existing Docker compose to be merged with provided docker run commands. I don't wanted to make it appear all time, so may be margin bottom, yes

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sharevb) [reply README.md:82]: added some text to say your are Lead Maintainer and me after

File: packages/composerize-website/src/components/Entry.js (hunk 1 above)
Comment (sharevb) [reply packages/composerize-website/src/components/Entry.js]: added margin bottom

**Pull request merged**